### PR TITLE
Image Request Feature added

### LINKED ISSUE #258: Feature: Image Request
## 🚀 Feature: Image Request 
(using python)

This script asks the user about the choice of selection from the avaliable catalogue of images.  After providing the respective choices , it will send request to the Photo API system for fetching the images and in case of desired user keywords (see #5) this is again ask the user about the keywords seperated by commas ( the commas will be cleared out using replace function)
and then fetch the images having similar keywords. The image will be displayed in default photo viewer of that OS.

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?
YES

 Please assign me on this issue
@GSSOC22'
GSSOC 2022

### PR DESCRIPTION
**Pull request by AlapanDas**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

YES I HAVE READ .

### Description

This feature helps the user to fetch images from the user provided keywords or random images from the unsplash library.

Fixes #258 

### Checklist
- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Working:
With Random Image---
image
With User Provided Keywords--
image

FUNFACT: This feature doesn't requires any browser activity , and the file is stored safely in a seperate image file (.jpg format)

### CODE REVIEW

**Comment by photon149** [comment]
@AlapanDas Please Read Contributing Guidelines while making the PR. Also , a few pointers which need to be addressed  :-
- Mention the Issue Number to which the particular PR relates.
- Enhance Code Readability (Add Comments)
- Add a README file and give a brief description of your project (including Output SS)

**Comment by AlapanDas** [comment]
@photon149 please check the PR once more i have successfully processed the changes

**Review by photon149 — CHANGES_REQUESTED** [review]
@AlapanDas You need to create a new file in your directory called as  "README.md" in your subdirectory and add the description and SS there.

**Comment by AlapanDas** [comment]
Done !.

**Comment by photon149** [comment]
Just one Minor change , you don't require to Add the Issue number on your README file.
Just delete the part and it's good to go.
@prakhar3062 looks fine from my end , have a look

**Comment by AlapanDas** [comment]
okay brother , just a minute
Done!

**Review by prakhar3062 — APPROVED** [review]
okay from side

**Comment by AlapanDas** [comment]
Please add a gssoc level

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@AlapanDas Change the location of the folder your project should go under 
[Basic level](https://github.com/AlapanDas/Project-Guidance/tree/main/Desktop%20Application/Basic/Python).
And reduce the image size please make these changes ASAP.

**Comment by AlapanDas** [comment]
I am changing the folder, but bro the image size is dynamic , since the user can select 4k or 2k picture the file size will increase dynamically with user's input . The best i can do is upload a blank container.jpg file but if the user asks for a 4k file it will take 1-3mb and we cannot control it .

File: Desktop Application/Intermediate/Python/Image Request/Readme.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+### Description
+
+This feature helps the user to fetch images from the user provided keywords or random images from the unsplash library.
+
+<!-- Replace `issue_no` with the issue number which is fixed in this PR -->
+## Workflow:
+The script runs , and asks the user about the type of image. After entering the input the script fetches the desired image and stores it in the container file, and also opens it in default image viewer of that respective OS.
+## Working:
+With Random Image---
+![image](https://user-images.githubusercontent.com/71692310/156886857-4f6c1111-4473-4303-8242-7037ec69e111.png)
```
Comment (Kushal997-das) [inline Desktop Application/Intermediate/Python/Image Request/Readme.md:10]: Here you can reduce the image file . Give height or weight to image and you can easily reduce the size of the image .

File: Desktop Application/Intermediate/Python/Image Request/Readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+With User Provided Keywords--
+![image](https://user-images.githubusercontent.com/71692310/156886858-32cadc78-7a50-4db0-82fb-94df38e7195d.png)
```
Comment (Kushal997-das) [inline Desktop Application/Intermediate/Python/Image Request/Readme.md:12]: Reduce image size

**Review by Kushal997-das — COMMENTED** [review]
Reduce the image size.

**Comment by Kushal997-das** [comment]
> I am changing the folder, but bro the image size is dynamic , since the user can select 4k or 2k picture the file size will increase dynamically with user's input . The best i can do is upload a blank container.jpg file but if the user asks for a 4k file it will take 1-3mb and we cannot control it .

You can easily do man in your readme.md file doesn't matter it is in random/dynamic or not .

**Comment by AlapanDas** [comment]
accha in readme file okay okay i was thinking of the original file

**Comment by AlapanDas** [comment]
Done

**Review by Kushal997-das — APPROVED** [review]

**Pull request merged**